appveyor.yml

```diff
@@ -13,7 +13,7 @@ install:
   # Use http instead as a workaround
   # - gem update --system
   - gem sources -r https://rubygems.org/
-  - gem install bundler --no-ri --no-rdoc --source http://rubygems.org/
+  - gem install bundler --no-ri --no-rdoc --source http://rubygems.org/ -v '< 2'
```

Comment:
For now we only build against Ruby < 2.3.0 on Appveyor but we will need to remove this workaround once if we build against Ruby >= 2.3.0 (or at least condition to install either bundler < 2 or bundler >= 2)